src/components/project/dto/update-project.dto.ts

```diff
@@ -54,8 +53,8 @@ export abstract class UpdateProject {
   @DateField({ nullable: true })
   readonly estimatedSubmission?: CalendarDate | null;
 
-  @Field(() => ProjectStep, { nullable: true })
-  readonly step?: ProjectStep;
+  // @Field(() => ProjectStep, { nullable: true })
```

Comment:
What is the best way to deprecate the `UpdateProject.step`?

Reply:
Add `deprecatedReason` to the field options